modules/bump-version.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+if [[ $(__write_version) == false ]]; then
+    return 0
+fi
```

Comment:
@jaspernbrouwer hi, thanks for the feedback.

In our company we don't use `$VERSION_FILE` and always specify the version manually (via our bot). That's why I decided to disable bumping of the version :)

What do you think about another idea – removing the `$VERSION_FILE` completely before executing the `semverbump.sh`? By using this approach, we don't need to save / touch `$VERSION_FILE`, change the code in the `semverbump.sh` as it tries to use tags and fallbacks to `0.0.0` by default.